build-scripts/update_sds_to_scap_1_3.py

```diff
@@ -0,0 +1,99 @@
+
+from __future__ import print_function
+
+import sys
+from urllib.parse import urlparse
+
+import ssg.constants
+import ssg.xml
+ocil_ns = ssg.constants.ocil_namespace
+
+oval_ns = ssg.constants.oval_namespace
+xccdf_ns = ssg.constants.XCCDF12_NS
+ds_ns = ssg.constants.datastream_namespace
+xlink_ns = ssg.constants.xlink_namespace
+cat_ns = ssg.constants.cat_namespace
+ns = {'oval': oval_ns,
+      'xccdf': xccdf_ns,
+      'ds': ds_ns,
+      'xlink': xlink_ns,
+      'cat': cat_ns}
+
+component_ref_prefix = "#scap_org.open-scap_cref_"
+
+# Inspired by openscap ds_sds_mangle_filepath() function
+def mangle_path(path):
+    path = path.replace('/', '-')
+    path = path.replace('@', '-')
+    return path
+
+def move_patches_up_to_date_to_source_data_stream_component(datastreamtree):
+    ds_checklists = datastreamtree.find(".//ds:checklists", ns)
+
+    for component_ref in ds_checklists:
+        component_id = component_ref.get('{%s}href' % xlink_ns)
+        component_id = component_id[1:]
+
+        # Locate the <xccdf:check> of the <xccdf:Rule> with id security_patches_up_to_date
+        oval_check = datastreamtree.find(".//ds:component[@id='%s']//xccdf:Rule[@id='xccdf_org.ssgproject.content_rule_security_patches_up_to_date']/xccdf:check[@system='%s']" % (component_id, oval_ns), ns )
```

Comment:
Pyhton 2.6 doesn't support XPath attributes, you have to loop and filter manually:
```
for rule in rules:
    id_ = rule.get("id")
    if id_ == "xccdf_org.ssgproject.content_rule_security_patches_up_to_date":
        checks = ....
...
```

Reply:
That is unfortunate.
I have implemented the filtering manually.